### DISCUSSION TITLE
Accessing j_inode_flags in apfs

### DISCUSSION THREAD

**Discussion by nanosparks** [dyorgio/apfs-clone-checker#5] [General]
Shouldn't it be possible to get j_inode_flags and check the INODE_WAS_CLONED flag using code similar to that used by https://github.com/linux-apfs/apfsprogs/blob/master/apfsck/inode.c?

**Comment by dyorgio** [comment]
Hi @nanosparks, as documented on [README.md](https://github.com/dyorgio/apfs-clone-checker/blob/main/README.md#work-todo) it is possible, but all filesystem structure needs to be decoded (there is no fast index/way to get a specific file/inode information directly).
